helm/net-exporter-chart/templates/daemonset.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: net-exporter
-        image: quay.io/giantswarm/net-exporter:[[ .SHA ]]
+        image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         args:
```

Comment:
`giantswarm/net-exporter` has a hyphen. Do you need use `index` here?

Reply:
Ah no, it is in the value itself 
nevermind

Reply:
No its OK. Index is only needed if the key name has a hyphen.